main.nf

```diff
@@ -70,28 +71,21 @@ workflow {
 
 // Show help message if --help is run or (||) a required parameter (input) is not provided
 
-if ( params.help || params.input == false ){   
+if ( params.help || params.fq == false ){   
```

Comment:
Changed conditions of running the help message. Will run if fq flag not provided.

Reply:
Will also exit and not try to proceed.